__DEFINES/clothing.dm

```diff
@@ -13,7 +13,8 @@
 #define CANEXTINGUISH 		128
 #define CONTAINPLASMAMAN 	256
 #define IGNORE_LUBE			512
-#define MAGPULSE			1024 //prevents slipping in space, singulo pulling, etc
+#define MAGPULSE			1024 //prevents slipping in space, singulo pulling, etc 
+#define GENDERFIT			2048 //Toggles gender fitting so it appends _f for female mob icon.
```

Comment:
Maybe FEMALEGENDERFIT is better?

Reply:
I mean its technically possible to do the inverse and add a male fit to women stuff with it. since it just splits but I can do it if you want.